README.md

```diff
@@ -1,13 +1,13 @@
-# Mobilizeamerica::Client
+# MobilizeAmericaClient::Client
```

Comment:
I don't love the `XyzClient::Client` duplication, but it's been the approach we took on other gems as well (e.g.: `IdentityApiClient::Client`)